tools/crm_mon.c

```diff
@@ -89,7 +90,7 @@ static pcmk__supported_format_t formats[] = {
     PCMK__SUPPORTED_FORMAT_HTML,
     PCMK__SUPPORTED_FORMAT_NONE,
     PCMK__SUPPORTED_FORMAT_TEXT,
-    CRM_MON_SUPPORTED_FORMAT_XML,
+    TOOLS_SUPPORTED_FORMAT_XML,
```

Comment:
Yeah, that seems reasonable.  It might also be nice to break that big block of conditionals out into a key/value store up top, but the GHashTable docs specifically say not to use it to build a static table.

Reply:
Fixed.